include/seqan3/argument_parser/validators.hpp

```diff
@@ -485,6 +485,82 @@ struct default_validator
     }
 };
 
+//!\brief A helper struct to chain validators recursively via operator&()
+//!\ingroup validators
+template <validator_concept validator_lhs_type, validator_concept validator_rhs_type>
+    requires std::Same<typename validator_lhs_type::value_type, typename validator_rhs_type::value_type>
+struct validator_chain_adaptor
+{
+    //!\brief Type of value that are tested by each of the validators.
+    using value_type = typename validator_lhs_type::value_type;
+
+    /*!\brief Constructing from a vector.
+     * \param[in] lhs_ Some validator to be chained to rhs_.
+     * \param[in] rhs_ Another validator to be chained to lhs_.
+     */
+    validator_chain_adaptor(validator_lhs_type const & lhs_, validator_rhs_type const & rhs_) :
+        lhs{lhs_}, rhs{rhs_}
+    {}
+
+    /*!\brief Calls the operator() of each validator on the value cmp.
+     * \param[in] cmp The value to validate.
+     * \throws argument_parser_exception
+     *
+     * This function delegates to the validation of each of the chained validators
+     * by calling their operator() one after the other. The behaviour depends on
+     * the chained validators.
+     */
+    void operator()(value_type const & cmp) const
+    {
+        lhs(cmp);
+        rhs(cmp);
+    }
+
+    //!\brief Returns a message that can be appended to the (positional) options help page info.
+    std::string get_help_page_message() const
+    {
+        return detail::to_string(lhs.get_help_page_message(), " ", rhs.get_help_page_message());
```

Comment:
It is if the help message of the validator is properly formatted which must be relied on.
If I introduce a new line this may introduce wrong formatting when printing the help page because the there new lines are handled depending on the size of the terminal.

Reply:
okay!